app/helpers/merge_requests_helper.rb

```diff
@@ -2,10 +2,28 @@ module MergeRequestsHelper
   include GovukLinkHelper
   include GovukVisuallyHiddenHelper
 
-  def display_value_or_placeholder(value, placeholder = "You didn't answer this question")
+  def display_value_or_placeholder(value, placeholder = "No answer provided")
     value.presence || content_tag(:span, placeholder, class: "app-!-colour-muted")
   end
 
+  def details_prompt_link(page, merge_request)
+    govuk_link_to(merge_request_details_prompt(page), send("#{page}_merge_request_path", merge_request, referrer: "check_answers"), class: "govuk-link govuk-link--no-visited-state")
+  end
+
+  def merge_request_details_prompt(page)
+    messages = {
+      "existing_absorbing_organisation" => "Answer if absorbing organisation is already active",
+      "helpdesk_ticket" => "Enter helpdesk ticket number",
+      "merging_organisations" => "Add merging organisations",
+      "merge_date" => "Set merge date",
+    }
+    messages[page] || "Enter #{lowercase_first_letter(page.humanize)}"
+  end
```

Comment:
Unlike elsewhere in the app it was not the best solution to create the link text by adding a prefix infront of the page id and was best to custom write the entire message.